Assets/Scripts/Models/Character.cs

```diff
@@ -593,8 +593,8 @@ private void GetNewJob()
 
         if (needPercent > 50 && needPercent <= 100 && need.RestoreNeedFurn != null)
         {
-            IEnumerable<Furniture> furnitures = World.Current.furnitures.Where(furniture => furniture.ObjectType == need.RestoreNeedFurn.ObjectType && furniture.IsUsable() == true);
-            if (furnitures.Count() > 0)
+            List<Furniture> furnitures = World.Current.furnitures.Where(furniture => furniture.ObjectType == need.RestoreNeedFurn.ObjectType && furniture.IsUsable() == true).ToList();
+            if (furnitures.Any())
```

Comment:
If this is `List<>` it is better to use `.Count` property of list instead of LINQ method.